deploy-cluster-aws/awsdeploy.sh

```diff
@@ -116,7 +117,7 @@ fab upgrade_setuptools
 
 if [ "$WHAT_TO_DEPLOY" == "servers" ]; then
     # (Re)create the RethinkDB configuration file conf/rethinkdb.conf
-    python create_rethinkdb_conf.py
+    python create_rethinkdb_conf.py --bind-http-to-localhost $BIND_HTTP_TO_LOCALHOST
```

Comment:
Reading the environment variable would also work, but then `create_rethinkdb_conf.py` would have to check if it's set, and complain if it's not. I guess it's just a matter of taste, and I prefer being explicit.

I also used the same "Mr. Explicit" style with the calls to `launch_ec2_nodes.py` and `clusterize_confiles.py`.